internal/mock/mocks.go

```diff
@@ -101,40 +101,56 @@ func (t Repository) PushSignature(ctx context.Context, mediaType string, blob []
 	return ocispec.Descriptor{}, ocispec.Descriptor{}, nil
 }
 
+type PluginMock struct {
+	Metadata        proto.GetMetadataResponse
+	ExecuteResponse interface{}
```

Comment:
Given the change below, this should be:
```suggestion
	ExecuteResponse interface{}
	ExecuteError    error
```

Reply:
Updated.